static/beta/stage/navigation/subscriptions-navigation.json

```diff
@@ -9,8 +9,8 @@
       "href": "/subscriptions/overview"
     },
     {
-      "id": "subscriptionsInventory",
-      "appId": "subscriptionsInventory",
+      "id": "subscriptionInventory",
```

Comment:
@karelhala the ID is referenced in the services.json file. We have to fix that as well.

Reply:
Let's keep it as is, so we don't have to change everything everywhere.